src/api/spec/factories/bs_requests.rb

```diff
@@ -23,7 +23,20 @@
         request.commenter = user.login
       end
       if request.bs_request_actions.none?
-        request.bs_request_actions << create(:bs_request_action, type: evaluator.type, source_project: evaluator.source_project)
+        if evaluator.type
+          request.bs_request_actions << create(:bs_request_action,
+                                               type: evaluator.type,
```

Comment:
why not `evaluator.type || set_bugowner` instead the `else`? :thinking:

Reply:
I guess I will just require a type - and forbid to create a :bs_request

Reply:
I did just that now - having a default in the parent factory just doesn't work out cleanly, so I changed the examples to create a :set_bugowner_request explicitly. Cleaner code.